client/state/reader/conversations/schema.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	CONVERSATION_FOLLOW_STATUS_FOLLOWING,
+	CONVERSATION_FOLLOW_STATUS_NOT_FOLLOWING,
+	CONVERSATION_FOLLOW_STATUS_MUTING,
+} from './follow-status';
+
+/* eslint-disable quote-props */
```

Comment:
eslint trips up on 'enum' being an unquoted key, since it's a reserved word. 

I added the quotes, but Prettier has other ideas :)

Reply:
I updated this in our eslint config (https://github.com/Automattic/eslint-config-wpcalypso/commit/768458f1e839bd9b3c86b42302f04b5ccf16959a) but forgot to update wp-calypso's version :(.

I'll do that now

Reply:
seems like there was an issue in upgrading to eslint v4. and for some reason thats a peer dep of the newest eslint-config-wpcalypso (which makes no sense because we can't use it)